Projects/SvitlanaLN/Selenium/Test_Selenium.cs

```diff
@@ -0,0 +1,41 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System.Threading;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using System;
+using System.Collections.Generic;
+
+
+namespace Selenium
+{
+    [TestFixture]
+    public class Test_Selenium
+    {
+        private IWebDriver driver;
+        public enum BrowserType { Chrome, FireFox }
+
+        [TearDown]
+        public void AfterAllMethods()
+        {
+            driver.Quit();
+
+        }
+        [Test]
+        [Parallelizable]
+        [TestCase("sv_lugan@ukr.net", "!Qwerty123", BrowserType.Chrome)]      // must be true   
+        [TestCase("sv_lugan@ukr.net", "Wrong_password", BrowserType.Chrome)]  // must be false
+        [TestCase("sv_lugan@ukr.net", "!Qwerty123", BrowserType.FireFox)]      // must be true   
+        [TestCase("sv_lugan@ukr.net", "Wrong_password", BrowserType.FireFox)]  // must be false
+        public void Test_method(string usrn, string passw, BrowserType brow)
+        {
```

Comment:
it could be better to use covariant interfaces https://metanit.com/sharp/tutorial/3.27.php

Reply:
Done.
But I can not use the attribute  [Parallelizable]  because the part of tests are failing.
May be ChromeDriver and FirefoxDriver use the same cookies ?